numpy/_core/strings.py

```diff
@@ -150,3 +152,57 @@ def strip(x1, x2=None):
     if x2 is None:
         return _strip_whitespace(x1)
     return _strip_chars(x1, x2)
+
+
+def replace(x1, x2, x3, x4):
```

Comment:
Why do the arguments no longer have meaningful names? Especially `x4`.

I can see a case for trying to avoid inconsistencies, but I liked the effort in `char` to just stick with what python uses (`old, new, count, /`)

Reply:
That's a good conversation to have. I went for consistency with the rest of the namespace, but I'm obviously okay to replace those with meaningful names. @ngoldbaum What do you think?

Reply:
Note that I'm fine with moving the discussion outside of this PR!

Reply:
Good point! Let's update the argument names to match the names in the CPython API as much as possible. The first argument could be called `arr` or `a` (`defcharray.py` seems to prefer `a`, so that choice might be the most consistent if not the most clear).

Reply:
Just to be sure we don't forget, maybe open a new issue for the naming?